### DISCUSSION TITLE
Reduce the popup menus

### DISCUSSION THREAD

**Discussion by awaixjvd** [JunkFood02/Seal#659] [General]
A big thanks for the team to develop such a great app. It's simply a hassle free solution to download from anywhere.

I have a suggestion to make. Once the URL is pasted into the app and it fetches the information, then the next popup to "configure your download" should be put into settings section, same like quality selection panel. I have set the quality to be downloaded by default and it never popsup again. The same should be done to configure downloads popup.

**Comment by JunkFood02** [comment]
Settings - General - Turn off "Configure before download"

**Reply by awaixjvd** [reply]
I apologize, i didn't see it.

**Comment by Marco-9456** [comment]
Join our [Telegram group](https://t.me/seal_app) for a faster support in the feature.
